src/hb-ot-glyf-table.hh

```diff
@@ -1098,6 +1098,14 @@ struct glyf
 	  if (points[i].is_end_point)
 	    break;
 	}
+
+	/* Skip contours with less than 3 points */
+	if (contour_length < 3)
```

Comment:
A glyf contour with two non-coincident points is still an implicitly closed contour with length but no area. This can make a difference when stroking. It's an odd case, but I would hate to preclude it. In particular, it seems GDI/DirectWrite only skip single point contours and report two point contours (even if the points are coincident).

Reply:
Agree.

Reply:
Done, added Stroking.ttf also from https://skia-review.googlesource.com/c/skia/+/266945 that tests the difference. Thanks!